### ISSUE TITLE
Swift4.1 Support

### ISSUE THREAD

**Issue by beryu** [ra1028/VueFlux#24]
I use VueFlux with Carthage.
VueFlux has the error below after update Xcode to 9.3.

```
Module compiled with Swift 4.0.3 cannot be imported in Swift 4.1: /Path/to/VueFlux.framework/Modules/VueFlux.swiftmodule/x86_64.swiftmodule
```

Do you plan to update for this?
Thanks

**Comment by beryu** [comment]
Also, I try to build with Carthage's `--no-use-binaries` option but failed to build by error below.

```
While emitting IR SIL function "@_T07VueFlux8ExecutorV015MainThreadInnerC033_CA010519110FC26C1E89058BA722AAD8LLC7executeyyycF".
 for 'execute(_:)' at /Path/to/VueFlux/Executor.swift:62:9
```

I think it is Swift compiler's bug, but please fix it if you could.

**Comment by ra1028** [comment]
@beryu 
Thanks for the report!
Fixed by @naoto0n2 in #25. 
It seemed to be a bug in Swift compiler, so I just released workaround as tag 1.4.1.
